endgame_postprocessing/post_processing/file_util.py

```diff
@@ -106,3 +106,12 @@ def get_flat_regex(file_name_regex, input_dir, glob_expression="**/*.csv"):
             iu=file_match.group("iu_id"),
             file_path=f"{input_dir}/{file}",
         )
+
+def get_matching_csv(path: str, country_code: str, iu_number: str):
+    matching_values = glob.glob(os.path.join(path, f"*{country_code}*{iu_number}.csv"))
+    if len(matching_values) != 1:
+        raise Exception(
+            f"Expected exactly one file for {country_code}{iu_number}," +
+            f"found {len(matching_values)}"
+        )
+    return matching_values
```

Comment:
⛏️ Sorry having done this I think you should return `matching_values[0]` - the goal is that someone calling this function based only on the name wouldn't necessarily expect to receive a result that they have to iterate on.